test/run_readelf_tests.py

```diff
@@ -142,13 +145,33 @@ def prepare_lines(s):
         return False, 'Number of lines different: %s vs %s' % (
                 len(lines1), len(lines2))
 
+    # Numeric position of the View column in the output file, if parsing readelf..decodedline
+    # output, and the GNU readelf output contains the View column. Otherwise stays None.
+    view_col_position = None 
     for i in range(len(lines1)):
         if lines1[i].endswith('debug_line section:'):
             # .debug_line or .zdebug_line
             flag_in_debug_line_section = True
 
         # readelf spelling error for GNU property notes
         lines1[i] = lines1[i].replace('procesor-specific type', 'processor-specific type')
+        
+        # The view column position may change from CU to CU:
+        if view_col_position and lines1[i].startswith('cu:'):
```

Comment:
Should this be `view_col_position is not None`?

Not a huge fan of "nullable integers" :-) because in boolean contexts 0 and None behave similarly

Reply:
0 is impossible - short of drastic format changes in GNU readelf, the View column is never the first one. So it's realistically either None or a positive int. But I can fix, no big deal.